src/components/common/Table/styles.module.css

```diff
@@ -5,6 +5,11 @@
   justify-content: flex-start;
   max-width: 900px;
   overflow-x: auto;
+  margin-bottom: var(--space-1);
+}
+
+.gridRow:last-of-type {
+  margin-bottom: 0;
```

Comment:
This causes a notable jump when toggling advanced details. I suggest to invert it and use `margin-top` instead and target the first element instead of the last.